topic-operator/src/main/java/io/strimzi/operator/topic/Config.java

```diff
@@ -210,6 +211,9 @@ private Value(String key, Type<? extends T> type, boolean required) {
     /** Do we use old ZooKeeper based TopicStore */
     public static final Value<Boolean> USE_ZOOKEEPER_TOPIC_STORE = new Value<>(TC_USE_ZOOKEEPER_TOPIC_STORE, BOOLEAN, "false");
 
+    /** The version of Kafka */
+    public static final Value<String> KAFKA_VERSION = new Value<>(TC_KAFKA_VERSION, STRING, "");
```

Comment:
What happens when the version is not specified? Will it use the latest or something like that?

Reply:
I'd bet that `RuntimeException` `"Error reading from classpath resource /kafka--topic-config-model.json"` will be thrown.

Reply:
I guess I merged this too early because it completely breaks the backwards compatibility for the TO. Maybe we should revert it? Can't it simple use the latest Kafka version as default?